front/app/translations/admin/en.json

```diff
@@ -1352,6 +1352,7 @@
   "app.containers.admin.ContentBuilder.delete": "Delete",
   "app.containers.admin.ContentBuilder.error": "error",
   "app.containers.admin.ContentBuilder.errorMessage": "There is an error on {locale} content, please fix the issue to be able to save your changes",
+  "app.containers.admin.ContentBuilder.iframeDescription": "Display content from an external website on your page in an HTML iFrame.",
```

Comment:
This does not seem a part of this PR?

Reply:
What do you recommend? Should I remove the commit? It doesn't seem to be in Master

Reply:
It's strange that it was added by CI - I'd say it's fine to leave it - `en.json` is automatically generated so changing it manually is not something we normally do. If it happens again - it will perhaps be a sign to investigate further what's going on..